src/devices/Pn532/samples/Program.cs

```diff
@@ -75,7 +76,8 @@
 
     // To run tests, uncomment the next line
     // RunTests(pn532);
-    ReadMiFare(pn532);
+    // ReadMiFare(pn532);
```

Comment:
Can you uncomment the Mifare part and comment the GPIO one? I guess most people are interested in reading the Mifare cards.

Reply:
sure